test/hotspot/jtreg/ProblemList-zgc.txt

```diff
@@ -44,6 +44,3 @@ serviceability/sa/TestJmapCoreMetaspace.java                  8268722,8268636
 serviceability/sa/TestJhsdbJstackMixed.java                   8248912   generic-all
 serviceability/sa/ClhsdbPstack.java#process                   8248912   generic-all
 serviceability/sa/ClhsdbPstack.java#core                      8248912   generic-all
-
-serviceability/dcmd/gc/HeapDumpAllTest.java                   8274196   linux-all,windows-all
-serviceability/dcmd/gc/HeapDumpTest.java                      8274196   linux-all,windows-all
```

Comment:
Before you push, you'll need to do a merge from mainline and should un-ProblemList sun/tools/jmap/BasicJMapTest.java.